build.gradle

```diff
@@ -65,6 +65,23 @@ dependencies {
     // google core libs
     implementation "com.google.guava:guava:28.2-jre"
 
+    implementation 'ch.qos.logback:logback-classic:1.2.3'
+    implementation 'io.github.openfeign:feign-okhttp:11.0'
+    implementation 'io.github.openfeign:feign-jackson:11.0'
+    implementation 'io.github.openfeign:feign-slf4j:11.0'
+
+    // docker
+    implementation 'com.github.docker-java:docker-java:3.2.5'
+    implementation 'com.github.docker-java:docker-java-transport-httpclient5:3.2.5'
+
+    // test
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
```

Comment:
junit5, thanks :D

Reply:
We still have both, we'll need to migrate at some point.